packages/common/container-definitions/src/loader.ts

```diff
@@ -463,6 +464,19 @@ export interface IContainer extends IEventProvider<IContainerEvents>, IFluidRout
      * @alpha
      */
     forceReadonly?(readonly: boolean);
+
+    /**
+     * Exposes the root object / entryPoint of the container.
```

Comment:
the term root is overloaded as its possible to have multiple root datastores, i would just stick with entryPoint, or maybe default for the below reference